caddytls/client.go

```diff
@@ -60,6 +61,11 @@ var newACMEClient = func(config *Config, allowPrompts bool) (*ACMEClient, error)
 		return nil, fmt.Errorf("%s: insecure CA URL (HTTPS required)", caURL)
 	}
 
+	// Set timeout by replacing ACME's http.Client
+	if HTTPTimeout > 1 {
+		acme.HTTPClient = http.Client{Timeout: time.Duration(HTTPTimeout) * time.Second}
```

Comment:
This should probably be assigned only once, not ever time we create a new ACME client. @xenolf can you confirm when you have a chance?